docs/docsite/rst/core_index.rst

```diff
@@ -36,6 +36,7 @@ This documentation covers the version of ``ansible-core`` noted in the upper lef
    :caption: Ansible getting started
 
    getting_started/index
+   getting_started_ee/index
```

Comment:
@sivel @nitzmahone - this PR is adding a getting started with EEs to the Ansible docs. This is for two reasons:
1 - People still don't know what they are or how to use them/create them.
2 - The builder docs don't seem to be the correct location since this getting started goes beyond just builder.

This particular line adds this guide to the core docs as well. What are your thoughts? 

My own nickel is I agree it goes in both places since a core user would also benefit from understanding EEs, but this may not match your vision of the core docs.

Reply:
I'm +1000 to the content itself- huge thanks to all involved for getting the ball rolling! I've got some minor suggestions for the content, but IMO nothing that can't be handled with follow-on PRs.

My only thought about duplicating this sort of content in the versioned project docs is that it's directly opposed to the desired end state we're working toward. Specifically, that most "task-oriented" and "ecosystem-oriented" HOWTOs like this will live in the new shared community docs space (especially where they span multiple component projects). IIUC, as things start moving around to their final homes in those areas, the remaining kernel of core's version-scoped project reference docs will move to a new repo with a separate versioned "core project reference" sub-path on the published site to match the other component projects. 

With that in mind, we should probably have some rough guidelines around where new/moved content gets published so we're not making that end state harder to reach.

Reply:
+1 dropping this from the core index. I've also been thinking that core should probably have it's own getting started that doesn't duplicate the package docs.

Also, to my mind, this content does not exactly qualify as a "how to". It is more a tutorial that helps gain new skills by doing whereas a proper how to would help you put existing skills to use. I think it's tricky with EEs because this feels like a how to because you need to use different tooling, a collection, and so on.

We don't have the right home for that content just yet and don't want to throw this EE tutorial up somewhere and increase fragmentation. It's also better, imho, not to promote this as a how to because, while this is a step in the right direction, we're not there just yet. So getting started seems like the best fit within the existing content set.

I feel like what we should do with this is publish to the package docs "devel" branch only and give the content some community soak time. In the meantime we can work on standing up a place for the How To content to live. A good option might be to create an ``ansible/ecosystem-documentation`` repository that deploys to the main project on RTD.